lang/resources/org/partiql/type-domains/partiql.ion

```diff
@@ -137,6 +143,12 @@
             // For `<expr> [AS <id>]`
             (project_expr expr::expr as_alias::(? symbol)))
 
+        // A list of LET bindings
+        (product let let_bindings::(* let_binding 1))
+
+        // A LET binding
+        (product let_binding expr::expr name::symbol)
```

Comment:
I would name this `as_alias::symbol` instead of `name::symbol` as it resembles the grammar more closely.

Reply:
What grammar are you referring to? The general [spec](https://github.com/partiql/partiql-spec/issues/15) uses the term `name`.

I kind of like `name` because the meaning is slightly different than `as_alias`, which is an optional parameter.

Reply:
Will leave as `name` for now. This could change as the implementation and spec are more complete.